setup.py

```diff
@@ -27,7 +27,6 @@
     zip_safe=False,
     install_requires=INSTALL_REQUIRES,
     test_suite='testflinger_cli.tests',
-    tests_require=TEST_REQUIRES,
```

Comment:
You can remove the TEST_REQUIRES = [] line too.

Reply:
Thanks, I didn't see that one